### PR TITLE
Updated Project.ts

### LINKED ISSUE #131: [BUG] Wrong link for BattleSnake project
# Bug Report

## Describe the bug

The link for the BattleSnake project under `View project` redirects to `https://forum.nhcarrigan.com/`. This page however is unreachable.

## Expected behavior

Maybe these links should redirect as follows:
- `View project` -> https://play.battlesnake.com/u/nhcarrigan/becca-lyria/
- `View code` -> https://github.com/nhcarrigan/battlesnake

## Screenshots

image
image

### PR DESCRIPTION
**Pull request by shubhamayush12**
Changed the title and added required links to liveUrl and codeUrl

# Pull Request

## Related Issue:

Closes #131

### CODE REVIEW

File: src/assets/data/Projects.ts [hunk 1]
```diff
@@ -19,10 +19,10 @@ export const projects: ProjectInt[] = [
       'A Discord bot built on the discord.js library with TypeScript. Includes a database connection to MongoDB, multiple API calls, and a number of available features. Documentation page built with Angular.',
   },
   {
-    title: 'BattleSnake',
+    title: 'Battlesnake',
```
Comment (SashoStoichkov) [inline src/assets/data/Projects.ts:22]: Please don't change the title
```suggestion
    title: 'BattleSnake',
```

File: src/assets/data/Projects.ts [hunk 2]
```diff
@@ -19,10 +19,10 @@ export const projects: ProjectInt[] = [
       'A Discord bot built on the discord.js library with TypeScript. Includes a database connection to MongoDB, multiple API calls, and a number of available features. Documentation page built with Angular.',
   },
   {
-    title: 'BattleSnake',
+    title: 'Battlesnake',
     image: 'project2.png',
-    liveUrl: 'https://forum.nhcarrigan.com',
-    codeUrl: 'https://play.battlesnake.com/u/nhcarrigan/becca-lyria/',
+    liveUrl: 'https://play.battlesnake.com/u/nhcarrigan/becca-lyria/',
+    codeUrl: 'https://github.com/nhcarrigan/battlesnake',
```
Comment (SashoStoichkov) [inline src/assets/data/Projects.ts:25]: LGTM 👍🏻

**Review by SashoStoichkov — CHANGES_REQUESTED** [review]

**Comment by SashoStoichkov** [comment]
Also change the description to say `Closes #131`

**Comment by shubhamayush12** [comment]
Sorry I don't know how to change the description

**Review by nhcarrigan — APPROVED** [review]
Looks good to me. Thanks @SashoStoichkov for the help!

**Comment by nhcarrigan** [comment]
Congrats on your first pull request (PR).

Thank you for your contribution to the page. We are happy to accept these changes and look forward to future contributions.

**Comment by SashoStoichkov** [comment]
> Looks good to me. Thanks @SashoStoichkov for the help!

I'm glad to help :+1:

**Pull request merged**